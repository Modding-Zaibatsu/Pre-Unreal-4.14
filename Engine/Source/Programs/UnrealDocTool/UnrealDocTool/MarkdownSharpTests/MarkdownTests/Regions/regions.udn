[REGION:regionParam]
    Block region
[/REGION]

Paragraph between.

[REGION:none]
* item
* item
    * nested item
* item
    * nested item
[/REGION]

[REGION:none]
    * item
    * item
        * nested item
    * item
        * nested item
[/REGION]

[REGION:none]
* item
* item

        * nested item
* item

        * nested item
[/REGION]

[REGION:regionParam2]Single line region[/REGION]