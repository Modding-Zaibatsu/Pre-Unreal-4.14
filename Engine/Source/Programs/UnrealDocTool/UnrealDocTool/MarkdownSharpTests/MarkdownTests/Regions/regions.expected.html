<?xml version="1.0" encoding="UTF-8" ?>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en-US">
    <head>
    </head>
    <body>
        <div class="regionParam">
            <p>Block region</p>
        </div>

        <p>Paragraph between.</p>

        <div class="none">
            <ul>
                <li>
                    <p>item</p>
                </li>
                <li>
                    <p>item</p>
                    <ul>
                        <li>
                            <p>nested item</p>
                        </li>
                    </ul>
                </li>
                <li>
                    <p>item</p>
                    <ul>
                        <li>
                            <p>nested item</p>
                        </li>
                    </ul>
                </li>
            </ul>
        </div>

        <div class="none">
            <ul>
                <li>
                    <p>
                        item
                    </p>
                </li>
                <li>
                    <p>
                        item
                    </p>
                    <ul>
                        <li>
                            <p>
                                nested item
                            </p>
                        </li>
                    </ul>
                </li>
                <li>
                    <p>item</p>
                    <ul>
                        <li>
                            <p>
                                nested item
                            </p>
                        </li>
                    </ul>
                </li>
            </ul>
        </div>

        <div class="none">
            <ul>
                <li>
                    <p>
                        item
                    </p>
                </li>
                <li>
                    <p>item</p>
                    <pre class="prettyprint"><code>* nested item</code></pre>
                </li>
                <li>
                    <p>item</p>
                    <pre class="prettyprint"><code>* nested item</code></pre>
                </li>
            </ul>
        </div>
        
        <p>
            <div class="regionParam2">
                <p>Single line region</p>
            </div>
        </p>
    </body>
</html>
